Common.props

```diff
@@ -1,8 +1,7 @@
 <Project>
   <PropertyGroup>
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
-    <SmoPackageVersion>150.18040.0-preview</SmoPackageVersion>
+    <SmoPackageVersion>140.17282.0-xplat</SmoPackageVersion>
```

Comment:
i think you can just set this to 140.17283.0 and not change the csproj files, so we keep using the unified package. If that still fails but using the separate xplat packages works, then we'll know there's something wrong with the distribution itself.

Reply:
@shueybubbles I'll try this.  I'm seeing an inconsistent repro now where sometimes the 15.0 binaries are able to get the completion suggestions and other times not.  

Though overall we have seen a big spike in IntelliSense issues opened in the past month, so I still think this is the most likely related change, but it doesn't look as clear cut as 14 works & 15 doesn't.  It needs more investigation, but if I can't root cause today I'll probably rollback to a "known good" pending further investigation.